packages/jazz-browser-auth-clerk/src/tests/BrowserClerkAuth.test.ts

```diff
@@ -0,0 +1,117 @@
+                mockLocalStorage[key] = value;
+            }),
+            removeItem: vi.fn((key: string) => {
+                delete mockLocalStorage[key];
+            }),
+            clear: vi.fn(),
+            length: 0,
+            key: vi.fn(),
+        };
+
+        // Mock Clerk client
+        mockClerkClient = {
+            user: {
+                unsafeMetadata: {},
+                fullName: "Test User",
+                username: "testuser",
+                id: "test-id",
+                update: vi.fn(),
+            },
+            signOut: vi.fn(),
+        };
+
+        // Mock driver
+        mockDriver = {
+            onError: vi.fn(),
+        };
+    });
+
+    describe("clerk credentials in localStorage", () => {
+        it("should get credentials from localStorage when clerk user is not signed in", async () => {
+            mockLocalStorage["jazz-clerk-auth"] = JSON.stringify({
+                accountID: "test-account-id",
+                accountSecret: "test-secret",
+            });
+
+            const auth = new BrowserClerkAuth(mockDriver, {
+                ...mockClerkClient,
+                user: null,
+            });
+
+            const result = await auth.start();
+            expect(result.type).toBe("existing");
+        });
+    });
+
+    describe("clerk credentials not in localStorage", () => {
+        it("should return new credentials when clerk user signs up", async () => {
+            const auth = new BrowserClerkAuth(mockDriver, mockClerkClient);
+            const result = await auth.start();
+            expect(result.type).toBe("new");
+        });
+
+        it("should return existing credentials when clerk user is signed in", async () => {
+            mockClerkClient = {
+                user: {
+                    unsafeMetadata: {
+                        jazzAccountID: "test-account-id",
+                        jazzAccountSecret: "test-secret",
+                    },
+                    fullName: "Test User",
+                    username: "testuser",
+                    id: "test-id",
+                    update: vi.fn(),
+                },
+                signOut: vi.fn(),
+            };
+
+            const auth = new BrowserClerkAuth(mockDriver, mockClerkClient);
+            const result = await auth.start();
+            expect(result.type).toBe("existing");
+        });
+
+        it("should throw error when not signed in", async () => {
+            const auth = new BrowserClerkAuth(mockDriver, {
+                ...mockClerkClient,
+                user: null,
+            });
+
+            await expect(auth.start()).rejects.toThrow("Not signed in");
+        });
+
+        it("should save credentials to localStorage", async () => {
+            const auth = new BrowserClerkAuth(mockDriver, mockClerkClient);
+            const result = await auth.start();
+            result.saveCredentials?.({
+                accountID: "test-account-id" as ID<Account>,
```

Comment:
**note:** the linter is breaking the build because of this error

Reply:
Good spot, fixed